app/src/main/java/org/fossasia/openevent/general/event/EventDetailsViewModel.kt

```diff
@@ -48,9 +48,11 @@ class EventDetailsViewModel(private val eventService: EventService) : ViewModel(
 
     fun loadMap(event: Event): String {
         // location handling
-        val BASE_URL = "https://api.mapbox.com/v4/mapbox.emerald/pin-l-marker+673ab7"
-        val LOCATION = "(" + event.longitude + "," + event.latitude + ")/" + event.longitude + "," + event.latitude
-        return BASE_URL + LOCATION + ",15/900x500.png?access_token=" + MAPBOX_KEY
+        val MAP_BASE_URL = "http://maps.google.com/maps/api/staticmap?center="
+        val MAP_LOCATION = "${event.latitude},${event.longitude}"
+        val MAP_SETTINGS = "&zoom=15&size=320x200&sensor=false"
+        val MAP_KEY = "&key = $GOOGLE_MAP_KEY"
+        return MAP_BASE_URL + MAP_LOCATION + MAP_SETTINGS + MAP_KEY
```

Comment:
This could remain as MapBox is better for free projects

Reply:
I have returned it back to Mapbox